README.md

```diff
@@ -0,0 +1,19 @@
+# Weaviate Data Loading Demos
+
+This repository contains Jupyter notebooks that illustrate various methods for loading data into a Weaviate database 
+using the [dlt](https://github.com/dlt-hub/dlt) library.
+
+## Repository Contents
+
+- [demo_weaviate.ipynb](https://github.com/dlt-hub/weaviate_demo/blob/main/demo_weaviate.ipynb): shows how to load data from PDF files, specifically invoices, into Weaviate.
+- [sql_to_weaviate.ipynb](https://github.com/dlt-hub/weaviate_demo/blob/main/sql_to_weaviate.ipynb): shows how to import data from a public MySQL database into Weaviate.
```

Comment:
can you add zendesk demo colab (you can convert it to JN)

Reply:
and we can add info how to open JN from github in colab https://colab.research.google.com/github/googlecolab/colabtools/blob/master/notebooks/colab-github-demo.ipynb

Reply:
I'll add it in a next PR.